File: Android/LearnDagger/app/src/main/java/com/srithar/learndagger/MainActivity.java
package com.srithar.learndagger;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import javax.inject.Inject;
import javax.inject.Singleton;


public class MainActivity extends AppCompatActivity {

    @Inject
    Laptop mLaptop;
    @Inject
    HardDrive mHardDrive;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        LaptopComponent laptopComponent=DaggerLaptopComponent.create();
        laptopComponent.inject(this);
        laptopComponent.getHardDrive();
        mLaptop.createLaptop();


    }
}

File: Android/RestartusingKotlin/app/src/main/java/com/example/restartusingkotlin/MainActivity.kt
package com.example.restartusingkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        val name: String = "Adam"
        Log.d("HelloMsg",(0 until 10)::class.simpleName.toString())

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }


}